tests/utils/gossamer_utils.go

```diff
@@ -119,6 +119,7 @@ func startGossamer(t *testing.T, node Node, websocket bool) (
 		"--rpcport", node.RPCPort,
 		"--rpcmods", "system,author,chain,state,dev,rpc",
 		"--rpc",
+		"--rpc-unsafe",
```

Comment:
Do we need this flag for end to end tests?

Reply:
yes, we call the `author_submitExtrinsic` which is unsafe